recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}   # required for C++ headers
+    - cmake
+    - make
+    - maven
+    - openjdk 8.*
+    - wget
+    - clangdev >=6.*
+    - go
+    - bisonpp
+    - flex
+  host:
+    - snappy  # required for mapd-core > 4.5.0
+    - arrow-cpp >=0.12
+    - blosc
+    - boost-cpp >=1.68.*
+    - double-conversion
+    - libgdal
+    - gflags
+    - glog
+    - libarchive
+    - libkml
+    - libpng
+    - llvmdev >=6.*
+    - openssl
+    - parquet-cpp
+    - thrift-cpp >=0.11.0
+  run:
+    - boost-cpp >=1.68.*
```

Comment:
I believe we have a global pinning for `boost-cpp`.

Reply:
I think this req is coming from the package itself, though, so this is OK

Reply:
Shouldn't that be `boost-cpp >=1.68` then?

Reply:
Uggh, yeah, probably

Reply:
fixed.